### PR TITLE
Add start scripts for documentation

### PR DESCRIPTION
**Pull request by polandll**
Please add this PR after Jeff Carpenter's PR. 

I've done the following in docker-images/examples:
- Added a start_***.sh that can be used to start docker-compose in a sane fashion.
- Added some variables for the docker image tagging (declared in the shell script, used in the docker-compose.yaml script).
- Added a .env file with the COMPOSE_PROJECT_NAME, that eases running multiple docker-compose files on the same machine.
- Added a "add_zeppelin" directory that explains how to add zeppelin to any environment.
- Added a README. md to all example directories.

### CODE REVIEW

File: examples/cassandra-3.11/docker-compose.yml [hunk 1]
```diff
@@ -2,8 +2,8 @@ version: '2'
 
 services:
   backend-1:
-    image: cassandra:3.11.8
-    container_name: backend-1
+    image: cassandra:${CASSTAG}
+    #container_name: backend-1
```
Comment (dimas-b) [inline examples/cassandra-3.11/docker-compose.yml:6]: Should be commented-out line be removed perhaps?

Comment (dimas-b) [reply examples/cassandra-3.11/docker-compose.yml:6]: Why do we need to comment it out, by the way? :)

File: examples/add_zeppelin/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+To add zeppelin to any multi-container environment, add the following information:
+
+To `docker-compose.yaml`:
+
+```
+  zeppelin:
+    image: apache/zeppelin:${ZEPPTAG}
+    #container_name: zeppelin
```
Comment (dougwettlaufer) [inline examples/add_zeppelin/README.md:8]:
```suggestion
    container_name: zeppelin
```

Should this be uncommented?

File: examples/cassandra-3.11/docker-compose.yml [hunk 3]
```diff
@@ -53,10 +53,11 @@ services:
       - 8081:8081
       - 8082:8082
       - 8084:8084
+      - 8085:8085
```
Comment (dougwettlaufer) [inline examples/cassandra-3.11/docker-compose.yml:56]: This one should no longer be needed.

File: examples/cassandra-3.11/start_cass_311.sh [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+# Bring up the 2nd C* node
+
+#docker-compose up -f $COMPOSE_FILE -d backend-2
+#(docker-compose logs -f backend-2 &) | grep -q "is now part of the cluster"
+
+# Bring up the 3rd C* node
+
+#docker-compose up -f $COMPOSE_FILE -d backend-3
+#(docker-compose logs -f backend-3 &) | grep -q "is now part of the cluster"
```
Comment (dougwettlaufer) [inline examples/cassandra-3.11/start_cass_311.sh:22]: Why aren't we bringing up the other two?

File: examples/add_zeppelin/README.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+```
+# Wait until stargate is up before bringing up zeppelin
+
+echo ""
+echo "Waiting for stargate to start up..."
+while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' http://localhost:8082/health)" != "200" ]]; do
+    printf '.'
+    sleep 5
+done
```
Comment (dougwettlaufer) [inline examples/add_zeppelin/README.md:33]: We could just add this to the end of all the scripts. Its a nice default that makes sure stargate is ready before turning the cluster over plus it also means the only change for Zepplin will be adding that last line.

**Comment by dougwettlaufer** [comment]
Looks good. Just a few comments regarding commented lines and ports.

File: examples/cassandra-3.11/docker-compose.yml (hunk 1 above)
Comment (polandll) [reply examples/cassandra-3.11/docker-compose.yml]: I'll take them out. I first commented them out when I realized running multiple docker-compose files at the same time interfered with one another when the containers had the same names.

File: examples/add_zeppelin/README.md (hunk 2 above)
Comment (polandll) [reply examples/add_zeppelin/README.md]: fixed

File: examples/cassandra-3.11/start_cass_311.sh (hunk 4 above)
Comment (polandll) [reply examples/cassandra-3.11/start_cass_311.sh]: removed comments

File: examples/add_zeppelin/README.md (hunk 5 above)
Comment (polandll) [reply examples/add_zeppelin/README.md]: done

File: examples/cassandra-3.11/docker-compose.yml (hunk 3 above)
Comment (polandll) [reply examples/cassandra-3.11/docker-compose.yml]: fixed

File: examples/cassandra-4.0/docker-compose.yml [hunk 6]
```diff
@@ -39,10 +36,9 @@ services:
       - HEAP_NEWSIZE=128M
       - MAX_HEAP_SIZE=1024M
       - CASSANDRA_SEEDS=backend-1
-      - CASSANDRA_CLUSTER_NAME=backend
+      - CASSANDRA_CLUSTER_NAME=c4-${CASSTAG}-cluster
   stargate:
-    image: stargateio/stargate-4_0:v1.0.0
-    container_name: stargate
+    image: stargateio/stargate-3_11:${SGTAG}
```
Comment (dougwettlaufer) [inline examples/cassandra-4.0/docker-compose.yml:41]:
```suggestion
    image: stargateio/stargate-4_0:${SGTAG}
```

File: examples/cassandra-4.0/docker-compose.yml [hunk 7]
```diff
@@ -56,8 +52,8 @@ services:
     mem_limit: 2G
     environment:
       - JAVA_OPTS="-Xmx2G"
-      - CLUSTER_NAME=backend
-      - CLUSTER_VERSION=4.0
+      - CLUSTER_NAME=c4-${CASSTAG}-cluster
+      - CLUSTER_VERSION=3.11
```
Comment (dougwettlaufer) [inline examples/cassandra-4.0/docker-compose.yml:56]:
```suggestion
      - CLUSTER_VERSION=4.0
```

File: examples/cassandra-3.11/start_cass_311.sh [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+
+docker-compose up -d backend-1
+
+# Wait until the seed node is up before bringing up more nodes
+
+(docker-compose logs -f backend-1 &) | grep -q "Created default superuser role"
+
+# Bring up the 2nd C* node
+
+#docker-compose up -f $COMPOSE_FILE -d backend-2
```
Comment (dougwettlaufer) [inline examples/cassandra-3.11/start_cass_311.sh:16]:
```suggestion
docker-compose up -f $COMPOSE_FILE -d backend-2
```

File: examples/cassandra-3.11/start_cass_311.sh [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+(docker-compose logs -f backend-2 &) | grep -q "is now part of the cluster"
+
+# Bring up the 3rd C* node
+
+#docker-compose up -f $COMPOSE_FILE -d backend-3
```
Comment (dougwettlaufer) [inline examples/cassandra-3.11/start_cass_311.sh:21]:
```suggestion
docker-compose up -f $COMPOSE_FILE -d backend-3
```

File: examples/cassandra-4.0/start_cass40.sh [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+
+docker-compose up -d backend-1
+
+# Wait until the seed node is up before bringing up more nodes
+
+(docker-compose logs -f backend-1 &) | grep -q "Created default superuser role"
+
+# Bring up the 2nd C* node
+
+#docker-compose up -f $COMPOSE_FILE -d backend-2
```
Comment (dougwettlaufer) [inline examples/cassandra-4.0/start_cass40.sh:16]:
```suggestion
docker-compose up -f $COMPOSE_FILE -d backend-2
```

File: examples/cassandra-4.0/start_cass40.sh [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+(docker-compose logs -f backend-2 &) | grep -q "is now part of the cluster"
+
+# Bring up the 3rd C* node
+
+#docker-compose up -f $COMPOSE_FILE -d backend-3
```
Comment (dougwettlaufer) [inline examples/cassandra-4.0/start_cass40.sh:21]:
```suggestion
docker-compose up -f $COMPOSE_FILE -d backend-3
```

File: examples/dse-6.8/start_dse_68.sh [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+
+docker-compose up -d backend-1
+
+# Wait until the seed node is up before bringing up more nodes
+
+(docker-compose logs -f backend-1 &) | grep -q "Created default superuser role"
+
+# Bring up the 2nd C* node
+
+#docker-compose up -f $COMPOSE_FILE -d backend-2
```
Comment (dougwettlaufer) [inline examples/dse-6.8/start_dse_68.sh:16]:
```suggestion
docker-compose up -f $COMPOSE_FILE -d backend-2
```

File: examples/dse-6.8/start_dse_68.sh [hunk 13]
```diff
@@ -0,0 +1,35 @@
...
+(docker-compose logs -f backend-2 &) | grep -q "is now part of the cluster"
+
+# Bring up the 3rd C* node
+
+#docker-compose up -f $COMPOSE_FILE -d backend-3
```
Comment (dougwettlaufer) [inline examples/dse-6.8/start_dse_68.sh:21]:
```suggestion
docker-compose up -f $COMPOSE_FILE -d backend-3
```

**Review by dougwettlaufer — APPROVED** [review]

**Pull request merged**